packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/ReferencesButton.test.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Menu, Dropdown, Tooltip } from 'antd';
+
+import ReferencesButton from './ReferencesButton';
+import transformTraceData from '../../../model/transform-trace-data';
+import traceGenerator from '../../../demo/trace-generators';
+import ReferenceLink from './ReferenceLink';
+
+describe(ReferencesButton, () => {
+  const trace = transformTraceData(traceGenerator.trace({ numberOfSpans: 10 }));
+  const focusMock = jest.fn();
+  const oneReference = trace.spans[1].references;
+
+  const moreReferences = oneReference.slice();
+  moreReferences.push({
+    refType: 'CHILD_OF',
+    traceID: trace.traceID,
+    spanID: trace.spans[2].spanID,
+  });
+  const baseProps = {
```

Comment:
`uiFind`, `history`, `location`, and `trace` are not part of TReferencesButtonProps`